api/swagger/swagger-dev.json

```diff
@@ -32652,7 +32652,7 @@
                             "description": "Channels.",
                             "type": "array",
                             "items": {
-                              "description": "Channel represents a single Notification Channel.\n\nreserved channels\n pushover_config = 5;\n opsgenie_config = 7;\n victorops_config = 9;\n wechat_config = 10;",
+                              "description": "Channel represents a single Notification Channel.",
```

Comment:
There is no notifications in the swagger.json that this API is deprecated. It seems that we can add it here by using ` option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {deprecated: true}` option.
Please take a look: https://github.com/grpc-ecosystem/grpc-gateway/blob/f31ce88d9dbe898943943139ce726d7842582fa1/protoc-gen-openapiv2/options/openapiv2.proto#L106

Reply:
Right - though seems this option is only supported for RPCs and not for messages (so we can't mark specific messages/schemas as deprecated in swagger).